src/modules/managePosts.js

```diff
@@ -0,0 +1,23 @@
+import { appId } from './manageLikes.js';
+
+const baseUrl = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appId}/comments`;
+
+export const postComment = async (id, username, comment) => {
+  const postObj = {
+    item_id: id,
+    username,
+    comment,
+  };
+  const response = await fetch(baseUrl, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify(postObj),
+  });
+  return response;
+};
```

Comment:
- [ ] The app currently allows users to post comments without any form validation, meaning both fields can be empty. It would be nice to add some validation to those  inputs for a better user experience.

Reply:
Thank you,

Although we added the required attribute to `html `elements still we check the values before sending them to API.